prac_06/car.py

```diff
@@ -4,14 +4,19 @@
 class Car:
     """Represent a Car object."""
 
-    def __init__(self, fuel=0):
+    def __init__(self, name="", fuel=0):
         """Initialise a Car instance.
```

Comment:
Docstring references fuel but not name, could add a similar layout for name string